src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -932,6 +932,77 @@ class CreateTableStatementSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class CreateTableAsStatementSegment(BaseSegment):
+    """A `CREATE TABLE AS` statement.
+
+    As specified in https://www.postgresql.org/docs/13/sql-createtableas.html
+    """
+
+    type = "create_table_as_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        OneOf(
+            Sequence(
+                OneOf("GLOBAL", "LOCAL", optional=True),
+                Ref("TemporaryGrammar", optional=True),
+            ),
+            "UNLOGGED",
+            optional=True,
+        ),
+        "TABLE",
+        Ref("IfNotExistsGrammar", optional=True),
+        Ref("TableReferenceSegment"),
+        AnyNumberOf(
+            Sequence(
+                Bracketed(
+                    Delimited(
+                        Sequence(
+                            Ref("ColumnReferenceSegment"),
+                            AnyNumberOf(Ref("ColumnConstraintSegment")),
+                        ),
+                    ),
+                ),
+                optional=True,
+            ),
+            Sequence("USING", Ref("FunctionSegment"), optional=True),
+            OneOf(
+                Sequence(
+                    "WITH",
+                    Bracketed(
+                        AnyNumberOf(
+                            Sequence(
+                                Ref("ParameterNameSegment"),
+                                Sequence(
```

Comment:
This second nested sequence isn't needed, just put those terms in the one above

Reply:
@WittierDinosaur This is also replicating what exists in the `CreateTableSegmentStatement`. I can remove it but the logic is the same according to the documentation.

Reply:
It's probably unnecessary if CreateTable too. For clarity, I'm just talking about de-nesting the inner sequence, not getting rid of what it contains. Bring the EqualsSegment and LiteralGrammar to the same level as ParameterNameSegment

Reply:
It should be removed from both now. It didn't seem to have any affect on the tests.